pkg/controllers/vdb/podfacts.go

```diff
@@ -1037,3 +1055,12 @@ func getHostList(podList []*PodFact) []string {
 	}
 	return hostList
 }
+
+// needAgentKeysCopy returns true if all agent keys are present in /home/dbadmin/share
+// and have not yet been copied to /opt/vertica/config/
+func needAgentKeysCopy(pod *PodFact) bool {
```

Comment:
Lets make this function part of `PodFact`:
```suggestion
func (p *PodFact) needAgentKeysCopy() bool {
```